snap/gui/fifteenpuzzle.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Game of Fifteen
+Comment=When the gong goes bong, stop talking!
```

Comment:
This is probably a mistake :D

Reply:
Whoops, yes copy and paste bug.  I'll fix